### PR TITLE
[timetagger] New guide for timetagger

### PR DESCRIPTION
**Pull request by Weishaupt**
This is a new guide for [timetagger](https://timetagger.app)

### CODE REVIEW

File: source/guide_timetagger.rst [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+To configure Timetagger we need to create our own main file.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ cp ~/.local/lib/python3.11/site-packages/timetagger/__main__.py ~/bin/mytimetagger.py
+ [isabell@stardust ~]$
+
+Next we select a random port number (e.g. 34423) and map it in the backend.
```
Comment (EV21) [inline source/guide_timetagger.rst:75]: I suggest to use a simple and fixed port number like 3000

**Review by EV21 — CHANGES_REQUESTED** [review]
Hi 👋,
I left a few comments.

File: source/guide_timetagger.rst [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+============
+
+To install timetagger we use :manual_anchor:`pip <lang-python.html#pip>`.
+
+.. note:: The ``pip``-command defaults to pip from python 2.7. Please use a modern version of pip instead. See the :manual:`manual <lang-python/#update-policy>` for currently supported versions. In this guide, we use python3.11.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ pip3.11 install --user -U timetagger
```
[the remark this answers is not in the document]
Comment (SalocinHB) [reply source/guide_timetagger.rst:53]: Doesn't hurt either if it's a second attempt, etc.

File: source/guide_timetagger.rst [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Installation
+============
+
+To install timetagger we use :manual_anchor:`pip <lang-python.html#pip>`.
+
+.. note:: The ``pip``-command defaults to pip from python 2.7. Please use a modern version of pip instead. See the :manual:`manual <lang-python/#update-policy>` for currently supported versions. In this guide, we use python3.11.
```
[the remark this answers is not in the document]
Comment (Weishaupt) [reply source/guide_timetagger.rst:48]: Timetagger fails to start with pyhon3.6, so I'm using version 3.10 instead.

    AttributeError: module 'asyncio' has no attribute 'run'

File: source/guide_timetagger.rst [hunk 4]
```diff
@@ -0,0 +1,171 @@
+.. highlight:: console
+
```
[the remark this answers is not in the document]
Comment (Weishaupt) [reply source/guide_timetagger.rst:2]: Thanks. All fixed.

File: source/guide_timetagger.rst (hunk 1 above)
Comment (Weishaupt) [reply source/guide_timetagger.rst]: Port 4800 it is. :-)

File: source/guide_timetagger.rst [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+=======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Updating is as simple as running pip again to update to the latest version and restarting the deamon.
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ pip3.11 install --user -U timetagger
```
[the remark this answers is not in the document]
Comment (Weishaupt) [reply source/guide_timetagger.rst]: Done.

File: source/guide_timetagger.rst [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+ […]
+ [isabell@stardust ~]$ supervisorctl restart timetagger
+ timetagger: stopped
+ timetagger: started
+ [isabell@stardust ~]$
+
+.. _Timetagger: https://timetagger.app/
+.. _feed: https://pypi.org/rss/project/timetagger/releases.xml
+.. _GPLv3 License: https://opensource.org/licenses/GPL-3.0
+.. _LICENSE: https://raw.githubusercontent.com/almarklein/timetagger/main/LICENSE
```
[the remark this answers is not in the document]
Comment (Weishaupt) [reply source/guide_timetagger.rst]: I've added a note in the beginning.

File: source/guide_timetagger.rst (hunk 4 above)
[the remark this answers is not in the document]
Comment (EV21) [reply source/guide_timetagger.rst:2]: deamon -> daemon
You missed that one in line 123, 126 and 151

File: source/guide_timetagger.rst [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+-----------------
+Create the file ``~/etc/services.d/timetagger.ini`` with the following content:
+
+.. note:: Replace the port and credentials with the credentials you previously created. Also, make sure to enter the correct home directory instead of isabell.
+
+.. code-block:: ini
+ :emphasize-lines: 2
+
+ [program:timetagger]
+ command=python3.10 /home/isabell/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
```
Comment (EV21) [inline source/guide_timetagger.rst:135]: We can just use the home variable so we have fewer isabells here.
```suggestion
.. note:: Replace the port and credentials with the credentials you previously created.

.. code-block:: ini
 :emphasize-lines: 2

 [program:timetagger]
 command=python3.10 %(ENV_HOME)s/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
```

File: source/guide_timetagger.rst [hunk 8]
```diff
@@ -0,0 +1,175 @@
...
+
+.. code-block:: console
+ :emphasize-lines: 1
+
+ [isabell@stardust ~]$ python3.10 ~/bin/mytimetagger.py --bind=0.0.0.0:4800 --credentials='isabell:$2a$08$SskWkSrYJnXvlwLPU7OAlecoCxMDs5vMr1Egs6INiqq1a4ZcH3wBa'
+ […]
+ [INFO 2022-07-08 12:46:34] Collected 4 assets from /home/isabell/.local/lib/python3.10/site-packages/timetagger/pages/.
+ [INFO 2022-07-08 12:46:34] Server is starting up
+
+We should now be able to successfully login. Finally we setup a deamon to run TimeTagger automatically.
```
Comment (EV21) [inline source/guide_timetagger.rst:123]:
```suggestion
We should now be able to successfully login. Finally we setup a daemon to run TimeTagger automatically.
```

File: source/guide_timetagger.rst [hunk 9]
```diff
@@ -0,0 +1,175 @@
...
+
+Set up the deamon
```
Comment (EV21) [inline source/guide_timetagger.rst:125]:
```suggestion
Set up the daemon
```

File: source/guide_timetagger.rst [hunk 10]
```diff
@@ -0,0 +1,175 @@
...
+.. include:: includes/supervisord.rst
+
+If it's not in state ``RUNNING`` after 60 seconds, something went wrong.
+
+Updates
+=======
+
+.. note:: Check the update feed_ regularly to stay informed about the newest version.
+
+Updating is as simple as running pip again to update to the latest version and restarting the deamon.
```
Comment (EV21) [inline source/guide_timetagger.rst:151]:
```suggestion
Updating is as simple as running pip again to update to the latest version and restarting the daemon.
```

**Review by EV21 — CHANGES_REQUESTED** [review]

File: source/guide_timetagger.rst (hunk 7 above)
Comment (Weishaupt) [reply source/guide_timetagger.rst:135]: As the user must replace the credentials anyways I'd prefer to keep it simple for the average joe and have him just replace `isabell` twice.

Comment (SalocinHB) [reply source/guide_timetagger.rst:135]: Please use variables whenever possible

**Comment by nichtmax** [comment]
Thanks @EV21 for reviewing, you're a 💎!

**Pull request merged**